### DISCUSSION TITLE
GitHub Phishing Notification??

### DISCUSSION THREAD

**Discussion by ValchanOficial** [community/community#109375] [Discussions] [answered]
### Select Topic Area

Bug

### Body

I received this email from `support@github.com`, but I am not from GitHub Recruitment Team
The link points to this website: `https://auth.githubtalentcommunity.online/`
This is a Phishing

![image](https://github.com/community/community/assets/16228014/b64612e6-1096-4a71-9ed2-92e210f9f4b2)

**Comment by jubnl** [comment]
Take a look at that : https://github.com/orgs/community/discussions/109171

**Comment by ettaboyle** [comment, marked as answer]
We understand the inconvenience caused by these notifications. Our teams are currently working on addressing these unsolicited phishing notifications. We want to remind our users to continue to use [our abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam) to raise any abusive or suspicious activity.

We would like to bring to our users’ attention to the following:
- Please do not click any links or reply to these notifications. Please [report them](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam).
- [Authorizing an OAuth app](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/authorizing-oauth-apps) can expose your GitHub account and data to a third party.  
- GitHub’s recruitment process would never mention its users via issues/PRs or other public content.
- GitHub recommends you periodically [review your authorized OAuth apps](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/reviewing-your-authorized-oauth-apps). 
This spam activity targets and abuses GitHub’s mention and notification functionality.
- This is a phishing campaign and is not the result of a compromise of GitHub or its systems.
